local/src/server/verifier-server.cc

```diff
@@ -297,8 +302,8 @@ swoc::Errata
 ServerReplayFileHandler::txn_open(YAML::Node const &node)
 {
   LoadMutex.lock();
-  _txn._req._is_request = true;
-  _txn._rsp._is_response = true;
+  _txn._req.set_is_request();
```

Comment:
I can add that as well.

Rather than taking flags, I generally prefer separate setting functions. There is a `set_is_request` and `set_is_response` method. Are you saying you prefer taking a flag? I think in the past we've agreed on separate functions.

Reply:
I added overrides taking the protocol.